src/components/PageModal/Page.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import Icon from 'cozy-ui/react/Icon'
+import Modal, { ModalHeader, ModalContent } from 'cozy-ui/react/Modal'
+import palette from 'cozy-ui/react/palette'
+import styles from 'components/PageModal/Page.styl'
+import iconArrowLeft from 'assets/icons/icon-arrow-left.svg'
+
+const BackButton = ({ onClick }) => (
+  <button type="button" onClick={onClick} className={styles.Page__BackButton}>
+    <Icon icon={iconArrowLeft} width={16} color={palette.coolGrey} />
+  </button>
+)
+
+class Page extends Component {
```

Comment:
Add a docstring please :)

Reply:
The one from the readme is very good ;)

This component renders a `Modal` on tablet and desktop viewports, and a `Page`
on mobile viewports. A `Page` is a special kind of fullscreen `Modal` that
shows the mobile bottom navigation bar when it's visible.

I think we can move it here and it will appear in the styleguidist as well.